cmd/kpod/common.go

```diff
@@ -31,3 +32,30 @@ func getStore(c *cli.Context) (storage.Store, error) {
 	is.Transport.SetStore(store)
 	return store, nil
 }
+
+func getConfig(c *cli.Context) (*libkpod.Config, error) {
+	config := libkpod.DefaultConfig()
+	if c.GlobalIsSet("config") {
+		err := config.FromFile(c.String("config"))
+		if err != nil {
+			return config, err
+		}
+	}
+	if c.GlobalIsSet("root") {
```

Comment:
It looks like they're global flags for the kpod command itself, but I still think they should be kept here - the code for parsing them and the flags themselves should live in the same place